### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by iamprakkie**
Updated README to include steps for Ubuntu setup using brew.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,78 +17,159 @@ Please refer to the Amazon EKS Blueprints Quick Start [documentation site](https
...
 
 - Node version is a current stable node version 18.x.
 
-```bash
-$ node -v
-v18.12.1
-```
+    ```bash
+    $ node -v
+    v20.3.1
```
Comment (shapirov103) [inline README.md:124]: This is not the latest stable version. The current stable version is `v18.17.0`. (check the output of `sudo n stable`).

File: README.md [hunk 2]
```diff
@@ -17,78 +17,159 @@ Please refer to the Amazon EKS Blueprints Quick Start [documentation site](https
...
+    sudo apt update
+    ```
+
+1. **Install make**
+
+    ```bash
+    sudo apt install make
+    ```
+
+1. **Install Node.js and npm**
```
Comment (shapirov103) [inline README.md:40]: If you are installing brew on ubuntu, why do we need instructions to install node and npm manually? Should not that be done with `brew install node`?

**Review by shapirov103 — COMMENTED** [review]
@iamprakkie great improvement, thank you for that. Please see a couple of comments.

File: README.md (hunk 1 above)
Comment (iamprakkie) [reply README.md]: updated to stable version node@18 installation using brew

File: README.md (hunk 2 above)
Comment (iamprakkie) [reply README.md]: updated to stable version node@18 installation using brew.

File: README.md [hunk 3]
```diff
@@ -17,78 +17,203 @@ Please refer to the Amazon EKS Blueprints Quick Start [documentation site](https
...
+    ```bash
+    sudo yum update
+    ```
+
+1. **Install `make`**
+
+    ```bash
+    sudo yum install make
+    ```
+1. **Install `brew`** on ubuntu by following instructions as detailed in [docs.brew.sh](https://docs.brew.sh/Homebrew-on-Linux)
```
Comment (shapirov103) [inline README.md:39]: This section is for RHEL, it is confusing why we are calling out ubuntu. I assume this should state "On Linux/ CentOS/RHEL".

File: README.md [hunk 4]
```diff
@@ -17,78 +17,203 @@ Please refer to the Amazon EKS Blueprints Quick Start [documentation site](https
...
+    sudo apt update
+    ```
+
+1. **Install `make`**
+
+    ```bash
+    sudo apt install make
+    ```
+
+1. **Install `brew`** on ubuntu by following instructions as detailed in [docs.brew.sh](https://docs.brew.sh/Homebrew-on-Linux)
```
Comment (shapirov103) [inline README.md:97]: Minor: Ubuntu (capitalized)

**Review by shapirov103 — COMMENTED** [review]
@iamprakkie Looks great, I added a couple of minor comments. Please confirm you validated the instructions functionally on both added installation options.

File: README.md (hunk 3 above)
Comment (iamprakkie) [reply README.md]: Thanks for catch here. Removed 'ubuntu' reference here.

File: README.md (hunk 4 above)
Comment (iamprakkie) [reply README.md]: Removed 'ubuntu' reference here as well.

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**